vcr/assets/assets/default.index.yaml

```diff
@@ -5,20 +5,63 @@ indices:
   - name: index_issuer
     parts:
       - iriPath: ["https://www.w3.org/2018/credentials#issuer"]
+  - name: index_subject_organization
+    parts:
+      - iriPath: ["https://www.w3.org/2018/credentials#credentialSubject"]
+      - iriPath:
+          [
+            "https://www.w3.org/2018/credentials#credentialSubject",
+            "http://schema.org/organization",
+            "http://schema.org/legalname",
+          ]
+        tokenizer: whitespaceOrExact
+        transformer: cologne
+      - iriPath:
+          [
+            "https://www.w3.org/2018/credentials#credentialSubject",
+            "http://schema.org/organization",
+            "http://schema.org/city",
+          ]
```

Comment:
This index is for checking, so it should do an exact match meaning that both the tokenizer and transformer are not needed.